core/templates/services/utils.service.ts

```diff
@@ -66,10 +66,10 @@ export class UtilsService {
   // not empty or not. So, we cannot have any specific type defined for the
   // arguments and the arguments are given a generic type of Object.
   /**
-   * @param {Object} a - the first object to be compared.
-   * @param {Object} b - the second object to be compared.
-   * @return {boolean} - true if a is equivalent to b, false otherwise.
-   */
+  * @param {Object} a - the first object to be compared.
+  * @param {Object} b - the second object to be compared.
+  * @return {boolean} - true if a is equivalent to b, false otherwise.
+  */
   isEquivalent(a: Object, b: Object): boolean {
```

Comment:
Can you please check the call sites of this function to see if we can maybe make the type definition a bit more tight?

Reply:
I tried to find a way to set a more specific type.But that function is used in 8 files, in general it compares elements of type Object or other interfaces. For that reason I left them in type Object.